lldb/tools/lldb-dap/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "lldb-dap",
-	"displayName": "LLDB VSCode",
-	"version": "0.1.0",
+	"displayName": "LLDB DAP",
```

Comment:
This file has a mix of tabs and spaces of various sizes, should this be standardized one way or the other?